src/main/kotlin/no/nav/syfo/service/SykmeldingService.kt

```diff
@@ -134,61 +139,65 @@ class SykmeldingService(
+                                    receivedSykmelding,
+                                    session,
+                                    syfoserviceProducer,
+                                    receivedSykmelding.sykmelding.id,
+                                    healthInformation,
+                                    dokArkivClient,
+                                    journalpostId,
+                                    loggingMeta
                             )
                             Status.MANUAL_PROCESSING -> manuellBehandling(
-                                journalpostId = journalpostId,
-                                fnr = pasient.fnr,
-                                aktorId = pasient.aktorId,
-                                dokumentInfoId = dokumentInfoId, datoOpprettet = datoOpprettet,
-                                loggingMeta = loggingMeta,
-                                sykmeldingId = sykmeldingId,
-                                sykmelder = sykmelder,
-                                ocrFil = ocrFil,
-                                kafkaproducerPapirSmRegistering = kafkaproducerPapirSmRegistering,
-                                sm2013SmregistreringTopic = sm2013SmregistreringTopic,
-                                cluster = cluster,
-                                enhetId = behandlendeEnhetId
+                                    journalpostId = journalpostId,
+                                    fnr = pasient.fnr,
+                                    aktorId = pasient.aktorId,
+                                    dokumentInfoId = dokumentInfoId, datoOpprettet = datoOpprettet,
+                                    loggingMeta = loggingMeta,
+                                    sykmeldingId = sykmeldingId,
+                                    sykmelder = sykmelder,
+                                    ocrFil = ocrFil,
+                                    kafkaproducerPapirSmRegistering = kafkaproducerPapirSmRegistering,
+                                    sm2013SmregistreringTopic = sm2013SmregistreringTopic,
+                                    cluster = cluster,
+                                    enhetId = behandlendeEnhetId
                             )
                             else -> throw IllegalStateException("Ukjent status: ${validationResult.status} , Papirsykmeldinger kan kun ha ein av to typer statuser enten OK eller MANUAL_PROCESSING")
                         }
                         log.info("Sykmelding håndtert automatisk {}", fields(loggingMeta))
+
                         return
                     }
                 } catch (e: Exception) {
                     PAPIRSM_MAPPET.labels("feil").inc()
                     log.warn("Noe gikk galt ved mapping fra OCR til sykmeldingsformat: ${e.message}, {}", fields(loggingMeta))
                 }
+
+                // Fallback hvis OCR er null ELLER parsing av OCR til sykmeldingformat mislykkes
+                manuellBehandling(
```

Comment:
Denne har flyttet seg. Blir dette riktig..? 

Jeg er også usikker på om jeg forstår kommentaren. Hvis OCR mangler, vil ikke da dokumentinfoid også mangle..? :)

Reply:
Den har flyttet på seg fordi vi har introdusert en ny if-else, og vi ønsker å ha denne innenfor else-blokken. Jeg tror det er riktig

Kommentaren er for å skjønne hvorfor kallet til manuellBehandling() i det hele tatt kalles der nede. Manglende dokumentInfoId sjekker vi på linje 83, men safDokumentClient.hentDokument() på linje 89 returnerer null hvis den ikke klarer å unmarshalle ocrFilen. Derfor kan ocrFil være null, uavhengig av om dokumentInfoId er satt

Reply:
Vi kjørte lenge med en feil fordi denne hadde flyttet seg, og siden klassen er så stor er det ca umulig å se om det blir riktig eller ikke kun utifra en PR. Det er derfor jeg spør :)

Reply:
Ja, jeg skjønner godt at du spør, og takk for at du passer på! :)